app/pageutils.py

```diff
@@ -24,3 +24,7 @@ def get_page(request):
     page_number = 1
 
   return (page_number)
+
+
+def result_message(search_range, total_returned_records, microservice_title):
+  return f'{microservice_title} \nShowing {search_range.get("rangeLow",0)} to { total_returned_records if search_range.get("rangeHigh",0) > total_returned_records else search_range.get("rangeHigh",0) } of total {total_returned_records}'
```

Comment:
This line should probably be split over several lines, or into a few helper functions, because it's far too long.

Reply:
Have  altered so that the embedded singleline if statement ison the line before,  removing the bulk of the length